edweiss-app/components/notifs/NotifDisplay.tsx

```diff
@@ -0,0 +1,259 @@
+import ReactComponent from '@/constants/Component';
+
+import TView from '@/components/core/containers/TView';
+import TText from '@/components/core/TText';
+import { callFunction } from '@/config/firebase';
+import t from '@/config/i18config';
+import { Color } from '@/constants/Colors';
+import { iconSizes } from '@/constants/Sizes';
+import { IconType } from '@/constants/Style';
+import { timeInMS } from '@/constants/Time';
+import { default as NotifList } from '@/model/notifs';
+import { CourseID } from '@/model/school/courses';
+import { useRef } from 'react';
+import { Swipeable } from 'react-native-gesture-handler';
+import TTouchableOpacity from '../core/containers/TTouchableOpacity';
+import Icon from '../core/Icon';
+
+
+// Constants
+
+// Icons
+const icons = {
+    message: 'chatbubbles-outline',
+    quiz: 'help-circle-outline',
+    submission: 'clipboard-outline',
+    grade: 'school-outline',                // others:  trophy-outline / ribbon-outline / medal-outline
+    announcement: 'megaphone-outline',      // others:  warning-outline / alert-circle-outline / alert-outline
+    event: 'pizza-outline',                 // others:  information-circle-outline
+    meeting: 'people-outline',
+    group: 'people-circle-outline',         // others:  people-circle
+    post: 'create-outline',
+    comment: 'chatbubble-ellipses-outline', // others:  chatbubble-outline
+    like: 'heart-outline',                  // others:  heart-sharp / thumbs-up-outline
+    follow: 'person-add-outline',
+};
+
+const iconsColor = {
+    message: 'color11',
+    quiz: 'color5',
+    submission: 'color4',
+    grade: 'color17',
+    announcement: 'color1',
+    event: 'color9',
+    meeting: 'color12',
+    group: 'color15',
+    post: 'color13',
+    comment: 'color7',
+    like: 'color8',
+    follow: 'color3',
+};
+
+const iconsColorBackground = {
```

Comment:
You can write this in a constante file...

Reply:
Thank you, I've bring the modifications in my new commit, you can check if needed. I decided to put those constants in the model in notifs.ts